priv/repo/seeds.exs

```diff
@@ -20,6 +20,30 @@ Repo.insert! %Location{
  zip: "80205"
 }
 
+Repo.insert! %Location{
+ name: "Charlie's Nightclub Denver",
+ address: "900 E Colfax Ave",
+ city: "Denver",
+ state: "CO",
+ zip: "80218"
+}
+
+Repo.insert! %Location{
+ name: "Hamburger Mary's Denver",
+ address: "1336 E 17th Ave",
+ city: "Denver",
+ state: "CO",
+ zip: "80218"
+}
+
+Repo.insert! %Location{
+ name: "Cheer Up Charlies",
```

Comment:
Didn't expect to see this here. Love that place. One of the things I miss about Austin <3